### PR TITLE
`pyproject.toml` decisions for the Iris-grib leads

### PR DESCRIPTION
**Pull request by trexfeathers**
Ping @pp-mo

Ref SciTools/iris-grib#408

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ authors = [
     {name = "Iris-grib Contributors", email = "scitools.pub@gmail.com"}
 ]
 classifiers = [
-    "Development Status :: 5 - Production/Stable",
+    "Development Status :: 4 - Beta",
```
Comment (trexfeathers) [inline pyproject.toml]: Consider this more appropriate since the semantic version is still pre-v1.

File: pyproject.toml [hunk 2]
```diff
@@ -34,10 +34,13 @@ dynamic = [
     "readme",
     "version",
 ]
-description = "GRIB interface for Iris"
+description = "Functionality for converting between weather/climate datasets stored as GRIB files and SciTools-Iris Cubes"
```
Comment (trexfeathers) [inline pyproject.toml:37]: Based on the latest documentation in SciTools/iris-grib#402

File: pyproject.toml [hunk 3]
```diff
@@ -34,10 +34,13 @@ dynamic = [
...
 keywords = [
     "iris",
     "GRIB",
+    "data-analysis",
+    "earth-science",
+    "meteorology",
```
Comment (trexfeathers) [inline pyproject.toml:43]: A selection of keywords from the `pyproject.toml` in SciTools/iris ([link](https://github.com/SciTools/iris/blob/e26bfaf93dd06f82f195ee303960dc3ae03b1d09/pyproject.toml#L39-L50)).

File: pyproject.toml (hunk 1 above)
Comment (pp-mo) [reply pyproject.toml]: No I don't think this is the best choice.  Although we do retain the right to change without warning, there is no sense that this is "pre-release" software.

So, according to [this wikipedia entry](https://en.wikipedia.org/wiki/Software_release_life_cycle#Beta), 
"""
software is feature-complete but likely to contain several known or unknown bugs ... 
more bugs in it than completed software and speed or performance issues, and may still cause crashes or data loss ...
beta release ... is typically the first time that the software is available outside of the organization that developed it.
"""

I don't think any of that really matches the current status of iris-grib

**Review by pp-mo — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (trexfeathers) [reply pyproject.toml]: OK, solid decision, thanks!

Comment (trexfeathers) [reply pyproject.toml]: 43efb73c6cd869eb6d0c575aa75a5eb53bc5cf71

**Pull request merged**